src/Collection.php

```diff
@@ -936,6 +937,23 @@ public function updateOne($filter, $update, array $options = [])
         return $operation->execute($server);
     }
 
+    /*
+     * ChangeStream outline
+     *
+     * @see ChangeStream::__construct() for supported options
+     * @param array          $pipeline       List of pipeline operations
+     * @param array          $options        Command options
+     * @throws InvalidArgumentException for parameter/option parsing errors
+     */
+    public function watch(array $pipeline, array $options = [])
```

Comment:
I think we can also default `$pipeline` to `[]`, as we do with `$filter` arguments for some other CRUD methods. That only needs to happen on the Collection method. The operation class can require the user to specify an array on their own.